dominode-bootstrapper/dominode_bootstrapper/geonodeadmin.py

```diff
@@ -228,19 +267,24 @@ def bootstrap(
     internal_group_name = 'dominode-internal'
     with httpx.Client() as client:
         geonode_manager = GeoNodeManager(client, base_url, username, password)
+        geoserver_manager = GeoServerManager(client, base_url)
```

Comment:
As it stands it is not possible to reuse the same http client with the two managers because we are modifying the client's default headers inside `GeoServerManager.__init__()`.

My suggestion is to not modify the client's default headers in __init__ but rather pass them in each request